packages/components/src/components/Anchor/Anchor.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { HTMLAttributeAnchorTarget } from "react";
+import { Text } from "../../primitives/Text";
+
+export interface AnchorProps
+  extends Omit<React.HTMLAttributes<HTMLAnchorElement>, "color"> {
+  /** The contents of the anchor. Can be text or valid text related HTML, i.e. strong elements. */
+  children: NonNullable<React.ReactNode>;
+  /** Sets target to "_blank" and rel to "noreferrer noopener". */
+  isExternal?: boolean;
+  /** Defines the relationship between a linked resource and the current document. */
+  rel?: string;
+  /** Where to display the linked URL, as the name for a browsing context, i.e. A blank window. */
+  target?: HTMLAttributeAnchorTarget;
+  /** The link url */
+  href?: string;
```

Comment:
Don't we have these props from the `HTMLAnchorElement` ? 🤔

Reply:
I was expecting that, but for some reason, it isn't working without it.

Reply:
It was a wrong inheritance :grin: